examples/configs/includeFunctionWithArgs.json

```diff
@@ -0,0 +1,16 @@
+{
+  "lambdaARN": "arn:aws:lambda:ap-southeast-2:0000000000:function:your-lambda",
+  "powerValues": [384, 512, 768, 1024, 1280],
+  "num": 20,
+  "payload": [
+    {
+      "payload": {
+        "$$include1": "./payloadAdaptor.js",
+        "$$args1": ["brochure.pdf"]
```

Comment:
I like this implementation for injecting external JSON or binary files. But same for the ptx script (it should belong to the Lambda Power Tuning repo).

So I'd keep only the most basic config examples here.